webtraversallibrary/screenshot.py

```diff
@@ -76,22 +76,21 @@ def capture(cls, name: str, driver: WebDriver, scale: float = 1.0, max_page_heig
         try:
             # We start from bottom, and go up!
             for shot_idx in range(num_shots, -1, -1):
-                # Select what elements to hide
+                # Don't hide elements at the top
                 if shot_idx == 0 or viewport.height > page_height:
-                    # No scrolling is needed, don't hide anything
                     js.show_position_fixed_elements(hidden_elements)
 
-                hidden = _scroll_to(js, shot_idx * int(viewport.height))
+                # Scroll, hide sticky elements, and take viewport screenshot
+                hidden = Screenshot._scroll_to(js, shot_idx * int(viewport.height))
                 hidden_elements.update(hidden)
-
                 viewport_screenshot = cls.capture_viewport("temp", driver, scale).image
 
                 if shot_idx * viewport.height > page_height - viewport.height:
                     new_viewport_area = (
                         0,
-                        viewport.height - (page_height - shot_idx * viewport.height),
+                        0,
                         viewport.width,
-                        viewport.height,
+                        page_height - shot_idx * viewport.height,
                     )
```

Comment:
Just to mention:
I'm not sure what the changes here mean because I don't know the context.